six/modules/c++/six.sidd/include/six/sidd/GeoTIFFWriteControl.h

```diff
@@ -59,12 +59,12 @@ class GeoTIFFWriteControl : public WriteControl
 public:
     GeoTIFFWriteControl();
     virtual ~GeoTIFFWriteControl() {}
-    
+
```

Comment:
Unrelated but go ahead and remove this destructor too... with good RAII design, we only expect destructors when something funky is going on, so having noop ones just make us pause for a sec for no reason